spec/features/idv/flow_spec.rb

```diff
@@ -173,6 +176,15 @@
       expect(page).to_not have_content(bad_phone_formatted)
     end
 
+    scenario 'failed attempt shows flash message' do
+      _user = sign_in_and_2fa_user
```

Comment:
if you're not going to use the `_user` variable, probably best just to ignore the return value.